packages/babel/test/fixtures/proposal-decorators/main.js

```diff
@@ -1,6 +1,8 @@
 @annotation
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default class MyClass {}
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
it's somewhat surprising that `@typescript-eslint` doesnt detect this as used - maybe worth updating those deps and if it doesn't help then an issue could be raised there about this

Reply:
I updated ESLint but the issue persisted, maybe I would really need to change ESLint config directly, but as this is just a test, maybe we leave it as it is.